apps.yaml

```diff
@@ -97,3 +97,15 @@ HELAO:
   categories:
     - technology-lab
     - technology-active-learning
+prisma:
+  metadata:
+    title: PRISMA
+    description: |
+        An app for for high-throughput analysis of spectra. More details in the app homepage and peer-reviewed article:
+    authors: Eibar Flores
+    external_url: https://github.com/BIG-MAP/PRISMA
+    logo: https://raw.githubusercontent.com/BIG-MAP/PRISMA/main/docs/figures/logo.png
+    state: development
+  categories:
+     - data analysis
```

Comment:
This category does currently not exist. You can add it to the [`categories.yml`](https://github.com/BIG-MAP/big-map-registry/blob/main/categories.yaml) if you like.

Reply:
I have now added the category to categories.yml